acousticbrainz/data/dataset.py

```diff
@@ -129,7 +131,7 @@ def get(id):
     try:
         connection = psycopg2.connect(current_app.config["PG_CONNECT"])
         cursor = connection.cursor()
```

Comment:
Is it worth using a dictcursor here so that we don't need to use `row[2]` later?